src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -169,14 +176,23 @@ public Server startServer() throws Exception {
     }
 
     private List<Service> getBeforeHealthCheckServices(Injector injector) {
-        List<Service> services = Stream.of(
+
```

Comment:
This is super nit-picky, totally up to your group on style, by why do you add an extra blank link at the start of some methods, but not others?

Reply:
Good catch :) I need to be consistent with this. When I think method is cluttered and not readable, I tend to group lines with blank line in between.